database/schema.sql

```diff
@@ -207,6 +207,72 @@ CREATE TABLE rotation_move_history (
   FOREIGN KEY (new_bucket) REFERENCES rotation_buckets (bucket_id)
 );
 
+-- BUDGET TABLES
+
+-- Table of all accounts Ruddock owns.
+CREATE TABLE budget_accounts (
+  account_id INTEGER NOT NULL AUTO_INCREMENT,
+  account_name VARCHAR(32) NOT NULL,
+  initial_balance NUMERIC(9,2) NOT NULL, -- this is sufficent to store up to 10 million
+  PRIMARY KEY (account_id)
+);
+
+-- Table of fiscal years and when they start.
+CREATE TABLE budget_fyears (
+  fyear_id INTEGER NOT NULL AUTO_INCREMENT,
+  fyear_num INTEGER NOT NULL, -- the number itself, like 2016
+  start_date DATE NOT NULL,
+  end_date DATE NOT NULL,
+  PRIMARY KEY (fyear_id)
+);
+
+-- Table of all budgets.
```

Comment:
Can you give an example of what would go in here? Or at least separately document what an account/budget/payee/etc is.